ignite/_utils.py

```diff
@@ -11,34 +11,19 @@ def _to_hours_mins_secs(time_taken):
     return hours, mins, secs
 
 
-def to_variable(batch, cuda=False, volatile=False):
-    if torch.is_tensor(batch):
+def to_tensor(variable, cuda=False, requires_grad=False):
+    if torch.is_tensor(variable):
         if cuda:
-            batch = batch.cuda()
-        return Variable(batch, volatile=volatile)
-    elif isinstance(batch, string_classes):
-        return batch
-    elif isinstance(batch, collections.Mapping):
-        return {k: to_variable(sample, cuda=cuda, volatile=volatile) for k, sample in batch.items()}
-    elif isinstance(batch, collections.Sequence):
-        return [to_variable(sample, cuda=cuda, volatile=volatile) for sample in batch]
-    else:
-        raise TypeError(("batch must contain tensors, numbers, dicts or lists; found {}"
-                         .format(type(batch[0]))))
-
-
-def to_tensor(variable, cpu=False):
-    if isinstance(variable, Variable):
-        t = variable.data
-        if cpu:
-            t = t.cpu()
-        return t
+            variable = variable.cuda()
```

Comment:
Wouldn't be better passing the device instead of cuda arg. in order to maintain this code agnostic to device? Then, this part of the code could be
```python
variable = variable.to(device)
```